src/test/responses/apim-get-api-404.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Nice

Reply:
Yes - this is one idea for some of the HTTP mocks where we could have real templated responses vs having to set it all up in code.